configs/graphql-compose.json

```diff
@@ -1,8 +1,8 @@
 {
   "index_name": "graphql-compose",
   "start_urls": [
-    "https://graphql-compose.github.io/docs/en/intro-quick-start.html",
-    "https://graphql-compose.github.io/docs"
+    "https://graphql-compose.github.io/docs/intro/quick-start.html",
+    "https://graphql-compose.github.io"
```

Comment:
Can you keep the "/docs" so that we only crawl documentation pages?

Reply:
Page `https://graphql-compose.github.io/docs` does not exists (404 code).
Is this ok that it in `start_urls`? It should be removed or it will be used as prefix check for urls?

I may revert last commit if need to remove `https://graphql-compose.github.io/docs`